packages/idealo.design/src/ui/components/Header/Header.jsx

```diff
@@ -42,28 +62,44 @@ class Search extends React.Component {
       margin: this.props.isOpen ? 'auto 2rem auto auto' : 0,
     }
 
+    const isLoggedIn = this.state.userInfo.status
+    this.state.isLoggedIn = isLoggedIn === 'LOGGED_IN';
```

Comment:
This won't work. You can only assign (by using `=`) to `this.state` in the constructor. At this place, you _have to_ use `this.setState()`.